csharp/nuget/Facebook.Yoga.Native.nuspec

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nuget spec file that will create a native only package for the Yoga.DLL...  This follows the pattern used by sqlite.native